Project.toml

```diff
@@ -26,6 +26,7 @@ NamedTupleTools = "d9ec5142-1e00-5aa0-9d6a-321866360f50"
 OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 ProgressLogging = "33c8b6b6-d38a-422a-b730-caa89a2f386c"
+PyFormattedStrings = "5f89f4a4-a228-4886-b223-c468a82ed5b9"
```

Comment:
We also depend on StrFormat.jl. I'm in favor of depending on at most one of these if possible. PyFormattedStrings is the better option imo as StrFormat hasn't been updated in a long time.
Would dropping StrFormat be a lot of work? I haven't worked on the code that uses it.

Reply:
Good catch! Of course I had forgotten all about StrFormat and StrLiterals 😸 
I agree with PyFormattedStrings.